frontend/tests/unit/tests/layout/query-menu-item-factory-test.js

```diff
@@ -75,7 +75,7 @@ describe('queryMenuItemFactory', function() {
     // The document does not seem to be cleaned up after each test instead each
     // test leaves additional DOM. Thus the tests are not independent.
     // Therefore we clean it by hand.
-    $document.find('body').html('');
+    menuContainer.remove();
```

Comment:
@furinvader @ulferts 

FYI: Karma really doesn't clean up the DOM so removing the element is fine. Clearing the body, on the other hand, will remove all custom assets (such as the CSS I required above), and thus fail other, completely unrelated tests.

We need to be careful what we remove. It should be fine to keep a reference to the element(s) we added and remove them afterwards.